app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -149,9 +157,7 @@
     private static final String EXTRA_CONSUMED_KEY = "login_extra_was_consumed";
 
     @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-
+    protected void onCreateAware(Bundle savedInstanceState) throws SessionUnavailableException {
```

Comment:
Overriding implementations of `onCreateAware` should probably still call their super implementation, to allow for more seamless extensibility in the future.